### DISCUSSION TITLE
Window lock function?

### DISCUSSION THREAD

**Discussion by IWORDI** [Chatterino/chatterino2#3258] [Ideas] [answered]
I might be the minority in this, but I was thinking it'd be cool if there was a 'window lock' button next to the settings wheel at the top of the window that would simply lock the size and location of the window. Because I typically want my chat in the same location, and don't want to accidentally move or resize it.. Adding a simple lock button at the top of the window would make it easy for anyone to lock/unlock the window for when they want to move or resize it.

Not sure if this is something that is do-able, but I just thought it was a neat idea and wanted to share.. :)

**Comment by LosFarmosCTL** [comment, marked as answer]
Even if it's possible, this sounds like a bad idea for chatterino to implement. I can understand and see the use case, but it's imo something that a window manager or similar program should handle, overwriting such an essential OS feature, to move windows around is never a good idea and is basically bound to create really annoying issues.

I would strongly suggest finding a tool that is purpose-built to doing something like this, instead of hacking together some half-baked crap inside of chatterino.
